engine/src/main/java/com/datatorrent/stram/plan/logical/LogicalPlanConfiguration.java

```diff
@@ -132,11 +881,11 @@ public static StramElement fromValue(String value) {
     @Override
     public Object fromString(String jsonObj)
     {
-      
+      LOG.debug("JONString {}", jsonObj);
       ObjectMapper mapper = ObjectMapperFactory.getOperatorValueDeserializer();
       try {
         return mapper.readValue(jsonObj, Object.class);
-      } catch (Exception e) {
+      } catch (IOException e) {
```

Comment:
A RuntimeException is thrown because this is a fatal error. The method cannot be modified to throw an IOException because that would be a backwards incompatible change for users of this class. This change is targeted for the next minor version release of Apex, so everything must be backwards and binary compatibly.